src/utils/session-storage-utils.ts

```diff
@@ -1,8 +1,11 @@
 export const getFromSessionStorage = (key: string): string | null => {
-  const item = sessionStorage.getItem(key);
+  const item = localStorage.getItem(key);
   return item ? JSON.parse(item) : null;
 };
 
-export const setToSessionStorage = (key: string, value: string | number | null): void => {
-  sessionStorage.setItem(key, JSON.stringify(value));
+export const setToSessionStorage = (
+  key: string,
+  value: string | number | null
+): void => {
+  localStorage.setItem(key, JSON.stringify(value));
```

Comment:
Please rename the function to reflect what it does (e.g. `setToLocalStorage`) as the current name is misleading.